buildpack.md

```diff
@@ -456,7 +469,104 @@ For each buildpack in the group, the lifecycle
 
 After analysis, the lifecycle MUST proceed to the build phase.
 
-## Phase #3: Build
+## Phase #3: Build-Ext (optional)
```

Comment:
This could potentially be moved to image-extension.md

Reply:
I'm +1 on that

Reply:
Updated in b677794